autopts/pybtp/defs.py

```diff
@@ -55,19 +55,20 @@ def BIT(bit):
 BTP_SERVICE_ID_PBP = 0x1e
 # GENERATOR append 1
 
+BTP_ERROR = 0x00
 BTP_STATUS_SUCCESS = 0x00
 BTP_STATUS_FAILED = 0x01
 BTP_STATUS_UNKNOWN_CMD = 0x02
 BTP_STATUS_NOT_READY = 0x03
 BTP_STATUS = 0x00
-CORE_READ_SUPPORTED_COMMANDS = 0x01
-CORE_READ_SUPPORTED_SERVICES = 0x02
-CORE_REGISTER_SERVICE = 0x03
-CORE_UNREGISTER_SERVICE = 0x04
-CORE_LOG_MESSAGE = 0x05
-CORE_EV_IUT_READY = 0x80
-GAP_READ_SUPPORTED_COMMANDS = 0x01
-GAP_READ_CONTROLLER_INDEX_LIST = 0x02
+BTP_CMD_CORE_READ_SUPPORTED_COMMANDS = 0x01
+BTP_CMD_CORE_READ_SUPPORTED_SERVICES = 0x02
+BTP_CMD_CORE_REGISTER_SERVICE = 0x03
+BTP_CMD_CORE_UNREGISTER_SERVICE = 0x04
+BTP_CMD_CORE_LOG_MESSAGE = 0x05
+BTP_EV_CORE_IUT_READY = 0x80
+BTP_CMD_GAP_READ_SUPPORTED_COMMANDS = 0x01
+BTP_CMD_GAP_READ_CONTROLLER_INDEX_LIST = 0x02
 GAP_SETTINGS_POWERED = 0
```

Comment:
Is the plan to also modify the remaining ones to be prefixed with BTP?

Reply:
Yes, I have finished modifying all services  and commands locally and will push changes in a matter of minutes

Reply:
Was referring to values such as `GAP_SETTINGS_POWERED` which do not have the BTP prefix

Reply:
For now only commands and events are refactored as this makes  btp header decoding easier.